test/scripts/selector-declaration-builder.test.js

```diff
@@ -0,0 +1,100 @@
+const $wDeclarationBuilder = require("../../scripts/selector-declaration-builder");
+
+describe("define $w.d.ts", () => {
```

Comment:
do we really need these tests ?
we usually try to write tests that verify the actual product, in our case, making sure the user gets correct autocomplete.
i'd prefer not to maintain unit tests that check the internals.

Reply:
Let's talk about this(and other question like work flows, the QA goal in our team etc.) in our offsite next week. WDYT?